script.js

```diff
@@ -486,6 +487,8 @@ function updateColumns() {
     }
   }
   columns = columnsEl.value;
```

Comment:
do we need to update bpm when we change the column count as well?

Reply:
Yes. Turns out the Steps object doesn't have a way to set time divisions. It just squeezes the full sequence in one measure.